src/components/Observations/UserCard.js

```diff
@@ -1,41 +1,22 @@
 // @flow
 
-import { useNavigation } from "@react-navigation/native";
-import UserIcon from "components/SharedComponents/UserIcon";
-import { Pressable, Text, View } from "components/styledComponents";
+import { Text, View } from "components/styledComponents";
 import type { Node } from "react";
 import React from "react";
 import { useTranslation } from "react-i18next";
-import IconMaterial from "react-native-vector-icons/MaterialIcons";
 import User from "realmModels/User";
 import useCurrentUser from "sharedHooks/useCurrentUser";
-import colors from "styles/tailwindColors";
 
 const UserCard = ( ): Node => {
-  const navigation = useNavigation( );
   const currentUser = useCurrentUser( );
   const { t } = useTranslation( );
-  if ( !currentUser ) { return <View className="flex-row mx-5 items-center" />; }
-  const navToUserProfile = ( ) => navigation.navigate( "UserProfile", { userId: currentUser.id } );
-
-  const uri = User.uri( currentUser );
 
   return (
-    <View className="flex-row px-5 items-center rounded-bl-3xl rounded-br-3xl bg-primary h-24">
-      {uri && <UserIcon uri={uri} />}
+    <View className="flex-row px-5 items-center h-24 bg-white">
       <View className="ml-3">
-        <Text className="color-white my-1">{User.userHandle( currentUser )}</Text>
-        <Text className="color-white my-1">
-          { t( "X-Observations", { count: currentUser.observations_count || 0 } )}
-        </Text>
+        <Text className="mt-1 text-3xl font-light">{t( "Welcome back," )}</Text>
```

Comment:
Yup, that translation string will need to accept the username as a variable, though if the username needs to be styled differently as it seems to here, I'm not yet sure how we do that in RN. On the web we could just use CSS, or we might use `ReactDOMServer.renderToString()` to render the formatted username to a string and pass that as a variable to the translation, but I doubt there's an analog in RN. Any ideas, anyone? If not, we can punt on it in this PR and just render the username the same as we do "Welcome back".

Reply:
I did create a substring-different-styled text component in Seek: [SubstringStyledText](https://github.com/inaturalist/SeekReactNative/blob/adfddb027faffb79eeb4abe08745bea35850d3c2/components/SeekYearInReview/SeekYearInReviewScreen.js#L45)

It works well for the UI side, but if I remember correctly it breaks the possibility to find this entire component in the unit test by text. The result looks like this:
<img width="355" alt="Screenshot 2023-01-31 at 13 28 45" src="https://user-images.githubusercontent.com/17345891/215760859-12238bf4-6e82-4533-ad17-e1c77eb605b0.png">

Reply:
`react-i18next` can handle this, at least on web. Not sure about native though https://react.i18next.com/latest/trans-component#usage-with-simple-html-elements-like-less-than-br-greater-than-and-others-v10.4.0

I can give it a try

Reply:
I sed an HTML renderer to handle this kind of thing in Seek, where the internationalized string would include HTML. Here's an example: [agree_to_terms](https://github.com/inaturalist/SeekReactNative/blob/adfddb027faffb79eeb4abe08745bea35850d3c2/translations/en.json#L125) is the string, and [here](https://github.com/inaturalist/SeekReactNative/blob/347397c30a073ea0b48ed1cf2bf6ffc19e8f70a1/components/Auth/Signup/LicensePhotosScreen.js#L171) is where it gets rendered in the code. In this case, I was using it to render links within a text string, but this can also be used to style specific html tags within a text string.